beta-20170420.md

```diff
@@ -0,0 +1,85 @@
+---
+title: What&#39;s New in beta-20170420
+toc: false
+summary: Additions and changes in CockroachDB version beta-20170420.
+---
+
+## Apr 20, 2017
+
+Get future release notes emailed to you:
+<div class="hubspot-install-form install-form-1 clearfix">
+    <script>
+        hbspt.forms.create({
+            css: '',
+            cssClass: 'install-form',
+            portalId: '1753393',
+            formId: '39686297-81d2-45e7-a73f-55a596a8d5ff',
+            formInstanceId: 1,
+            target: '.install-form-1'
+        });
+    </script>
+</div>
+
+### Binaries
+
+<div id="os-tabs" class="clearfix">
+    <a href="https://binaries.cockroachdb.com/cockroach-beta-20170420.darwin-10.9-amd64.tgz"><button id="mac" data-eventcategory="mac-binary-release-notes">Mac</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-beta-20170420.linux-amd64.tgz"><button id="linux" data-eventcategory="linux-binary-release-notes">Linux</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-beta-20170420.windows-6.2-amd64.exe.tgz"><button id="windows" data-eventcategory="windows-binary-release-notes">Windows</button></a>
+</div>
+
+### Backwards-Incompatible Changes
+
+- Security-related command-line arguments have been changed. Instead of individual flags, certificates and keys are now loaded from `--certs-dir` (defaults to `~/.cockroach-certs`, or settable with `COCKORACH_CERTS_DIR` environment variable). The individual flags still work in this release but will be removed in the next one. The `--insecure` flag is now *required* for insecure mode. [#14703](https://github.com/cockroachdb/cockroach/pull/14703)
+- The `--alsologtostderr` argument has been renamed to `--logtostderr`, and now takes an optional argument to specify a threshold. [#14841](https://github.com/cockroachdb/cockroach/pull/14841)
+- The `^` operator now means `pow()` instead of bitwise xor (this is consistent with PostgreSQL). [#14882](https://github.com/cockroachdb/cockroach/pull/14882)
```

Comment:
May need to add a note: `Any tables with ^ in a DEFAULT or CHECK expression will need to be changed to use # instead.`